daemon/api_general.go

```diff
@@ -130,13 +130,14 @@ func sysInfo(c *Command, r *http.Request, user *auth.UserState) Response {
 	}
 
 	m := map[string]interface{}{
-		"series":         release.Series,
-		"version":        c.d.Version,
-		"build-id":       buildID,
-		"os-release":     release.ReleaseInfo,
-		"on-classic":     release.OnClassic,
-		"managed":        len(users) > 0,
-		"kernel-version": osutil.KernelVersion(),
+		"series":          release.Series,
+		"version":         c.d.Version,
+		"build-id":        buildID,
+		"os-release":      release.ReleaseInfo,
```

Comment:
As there is a new JSON exported field in this struct, it'd be good to update `OSRelease` in `client/client.go` to decode it.

If we keep `on-core-desktop` below, it should also be exposed in `SysInfo` from that file too.